extensions/wikia/DesignSystem/ds.built.js

```diff
@@ -0,0 +1,768 @@
+	/******/ 	__webpack_require__.d = function(exports, name, getter) {
+		/******/ 		if(!__webpack_require__.o(exports, name)) {
+			/******/ 			Object.defineProperty(exports, name, {
+				/******/ 				configurable: false,
+				/******/ 				enumerable: true,
+				/******/ 				get: getter
+				/******/ 			});
+			/******/ 		}
+		/******/ 	};
+	/******/
+	/******/ 	// getDefaultExport function for compatibility with non-harmony modules
+	/******/ 	__webpack_require__.n = function(module) {
+		/******/ 		var getter = module && module.__esModule ?
+			/******/ 			function getDefault() { return module['default']; } :
+			/******/ 			function getModuleExports() { return module; };
+		/******/ 		__webpack_require__.d(getter, 'a', getter);
+		/******/ 		return getter;
+		/******/ 	};
+	/******/
+	/******/ 	// Object.prototype.hasOwnProperty.call
+	/******/ 	__webpack_require__.o = function(object, property) { return Object.prototype.hasOwnProperty.call(object, property); };
+	/******/
+	/******/ 	// __webpack_public_path__
+	/******/ 	__webpack_require__.p = "";
+	/******/
+	/******/ 	// Load entry module and return exports
+	/******/ 	return __webpack_require__(__webpack_require__.s = 10);
+	/******/ })
+/************************************************************************/
+/******/ ([
+	/* 0 */
+	/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+		"use strict";
+		Object.defineProperty(__webpack_exports__, "__esModule", { value: true });
+		/* harmony import */ var __WEBPACK_IMPORTED_MODULE_0_dropdown_dropdown_js__ = __webpack_require__(1);
+		/* harmony import */ var __WEBPACK_IMPORTED_MODULE_1_test_component_test_component_js__ = __webpack_require__(2);
+		/* harmony import */ var __WEBPACK_IMPORTED_MODULE_2_dropdown_dropdown_css__ = __webpack_require__(9);
+		/* harmony import */ var __WEBPACK_IMPORTED_MODULE_2_dropdown_dropdown_css___default = __webpack_require__.n(__WEBPACK_IMPORTED_MODULE_2_dropdown_dropdown_css__);
+		/* harmony reexport (module object) */ __webpack_require__.d(__webpack_exports__, "dropdown", function() { return __WEBPACK_IMPORTED_MODULE_0_dropdown_dropdown_js__; });
+		/* harmony reexport (module object) */ __webpack_require__.d(__webpack_exports__, "testComponent", function() { return __WEBPACK_IMPORTED_MODULE_1_test_component_test_component_js__; });
+
+
+
+		// if you want to add styles, just require them below and let Webpack do it's magic
+
+
+
+
+		/***/ }),
+	/* 1 */
+	/***/ (function(module, __webpack_exports__, __webpack_require__) {
+
+		"use strict";
+		Object.defineProperty(__webpack_exports__, "__esModule", { value: true });
+		/* harmony import */ var __WEBPACK_IMPORTED_MODULE_0__utils_closest_element__ = __webpack_require__(3);
+		/* harmony export (immutable) */ __webpack_exports__["hookDropdown"] = hookDropdown;
+
+
+		var dropdownClass = 'wds-dropdown';
+		var dropdownToggleClass = 'wds-dropdown__toggle';
+		var dropdownContentClass = 'wds-dropdown__content';
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'dropdownContentClass' variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "dropdownContentClass" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)